vue.config.js

```diff
@@ -52,15 +52,17 @@ const config = {
             media: 'https://sos.exo.io/c2corg-demov6-active',
             imageBackend: 'https://images.demov6.camptocamp.org',
             forum: 'https://forum.demov6.camptocamp.org',
-            readWrite: true
+            readWrite: true,
+            recaptchaKey: '6LfWUwoUAAAAAAxud1qqok6wOJJlCUsYXxHizRhc'
         },
         prod: {
             name: 'prod',
             api: 'https://api.camptocamp.org',
             media: 'https://media.camptocamp.org/c2corg_active',
             imageBackend: 'https://images.camptocamp.org',
             forum: 'https://forum.camptocamp.org',
-            readWrite: false // conservative approach : you have to explicity set it to true
+            readWrite: false, // conservative approach : you have to explicity set it to true
+            recaptchaKey: '6Lc9Cw4UAAAAAIKnlar0AOsGX_P5S-bk9u8viuo2'
```

Comment:
is it OK to store this key on the repo? it doesn't need to be hidden?

Reply:
yes, it's the public key send from UI to google server (that the way I've getted them :) )